srcopsmetrics/entities/commit.py

```diff
@@ -33,7 +33,7 @@ class Commit(Entity):
     entity_schema = Schema(
         {
             "pull_request": Any(None, int),
-            "patch": Schema({str: str}),
+            "patch": Schema({str: Any(None, str)}),
```

Comment:
I think you want `Optional[str]` in this case, as it includes str and None by default

Reply:
types other than those from `voluptuous` are not supported in schemes. There is also `Optional` in `voluptuous`, but it is intended for optional dictionary keys, not values. `Any` is intended to allow `None` values.
see https://github.com/alecthomas/voluptuous#allow-none-values